ARK Server Manager/ServerSettingsControl.xaml.cs

```diff
@@ -163,11 +163,12 @@ private void ReinitializeNetworkAdapters()
             preferredIP.Description = "(Recommended) " + preferredIP.Description;
             if(String.IsNullOrWhiteSpace(this.Settings.ServerIP))
             {
-                if(preferredIP != null)
-                {
-                    this.Settings.ServerIP = preferredIP.IPAddress;
-                }
-            } 
+                // removed to enforce the 'Let ARK choose' option.
```

Comment:
This is breaking behavior.  Whereas previously ASM would have been choosing the IP, now ARK will, and this may differ between machines from how it was before.  We should message this on the forums so people are not caught unaware when the update goes out.

An even better fix may be to add a specific entry to the drop down which is "Let ARK choose".  Then we can leave this behavior as-is, and people can opt in to the ARK behavior by specifically selecting that option from the drop down.